user_guides/jms/src/main/jbake/content/config.inc

```diff
@@ -1,21 +1,3 @@
-<!--
-
-    Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
-
-    This program and the accompanying materials are made available under the
-    terms of the Eclipse Public License v. 2.0, which is available at
-    http://www.eclipse.org/legal/epl-2.0.
-
-    This Source Code may also be made available under the following Secondary
-    Licenses when the conditions for such availability set forth in the
-    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
-    version 2 with the GNU Classpath Exception, which is available at
-    https://www.gnu.org/software/classpath/license.html.
-
-    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
-
--->
-
```

Comment:
@edbratt why is Copyright not needed for this file?  Thanks for explaining.

Reply:
Copyright is covered in the consolidated document. Regardless, this form of the copyright is an error since it renders in the final document which is just a mistake.